app/src/main/res/xml/preferences.xml

```diff
@@ -11,10 +11,11 @@
             android:icon="@drawable/ic_network_wifi_grey600_24dp"
             android:key="@string/pref_key_wifi_only"
             android:layout="@layout/item_pref"
-            android:title="@string/settings_wifi_only" />
+            android:title="@string/settings_wifi_only"
+            app:defaultValue="true" />
```

Comment:
Why did you change the default value for this setting? It should not be touched for fixing the issue.